src/assemble_workflow/README_RPM.md

```diff
@@ -0,0 +1,184 @@
+# OpenSearch/Dashboards DEB/RPM Distribution Design Document
```

Comment:
Design -> Documentation in README.md. Design documents get obsolete the day we make 1 change in the code, and are misleading. If you want to document the original intent, put it in a GitHub issue and add "See xyz for more information in the README."

Reply:
Thanks, #1452 is the issue and I have reduced the readme to have only the user-facing content.
Linked the issue.